shared/charts/couchdb/templates/statefulset.yaml

```diff
@@ -61,13 +61,29 @@ spec:
             - name: ERL_FLAGS
               value: "{{ range $k, $v := .Values.erlangFlags }} -{{ $k }} {{ $v }} {{ end }}"
           livenessProbe:
+{{- if .Values.couchdbConfig.chttpd.require_valid_user }}
```

Comment:
Why do we have the split behavior - is there any disadvantage to changing this to exec every time?
 
What I'm thinking is that while the result **_should_** be the same, why introduce the possibility of different behavior?

Reply:
I was afraid of having errors by using credentials when they are not needed. That's why I prefer to use the old check.